src/modules/playback-engine/types.ts

```diff
@@ -100,19 +96,33 @@ interface IPlaybackEngine {
   getDebugInfo(): IEngineDebugInfo;
 
   destroy(): void;
+
+  changeOutput(chromecastOutput: IVideoOutput): void;
+  resetOutput(): void;
 }
 
+type IEngineDebugInfo = INativeDebugInfo | ICromecastDebugInfo;
+
 /**
  * @property viewDimensions - Current size of view port provided by engine (right now - actual size of video tag)
  * @property currentTime - Current time of playback
  * @property duration - Duration of current video
  * @property loadingStateTimestamps - Object with time spend for different initial phases
+ * @property output - Type of the output (html5video, chromecast etc.);
  */
-interface IEngineDebugInfo extends IAdapterDebugInfo {
+interface INativeDebugInfo extends IAdapterDebugInfo {
```

Comment:
Why you put `IChromecastDebugInfo` here?

Reply:
Do you think, I should move it to `/src/modules/playback-engine/output/chromecast/types` ?

Reply:
yes, and made `IEngineDebugInfo` more generic, not a union type